Makefile

```diff
@@ -109,6 +109,9 @@ test-check-packages-with-custom-agent:
 test-build-zip:
 	./scripts/test-build-zip.sh
```

Comment:
What does this apart of just building the zip? :sweat_smile:

Reply:
This script builds the packages and then installs them (without the zip file). It would be briefly like:
```shell
elastic-package stack up -v -d

# for each package
elastic-package build -v --zip --sign 
elastic-package install -v

elastic-package stack down -v
```

Then, there is another script that builds the packages and installs them using the zip file (`scripts/test-install-zip.sh`):
```shell
elastic-package stack up -v -d

# for each package
elastic-package build -v --zip --sign 
elastic-package install -v --zip <zip_file_path>
# check via curl that it is installed

elastic-package stack down -v
```

And there is another script file, that just builds the zip file. Used specially for testing in Macos ARM:
```shell
elastic-package stack up -v -d

# for each package
elastic-package build -v --zip --sign 

elastic-package stack down -v
```

Probably, they could be renamed so it is clear what they do. What about rename them as this? @jsoriano

- `scripts/test-build.zip.sh`: `scripts/test-build-install.sh`
    - Package build and installed from the working copy 
- `scripts/test-install.zip.sh`: `scripts/test-build-install-zip-file.sh`
    - Package built and installed using the zip file
- `scripts/test-just-build-zip.sh`: `scripts/test-build.zip.sh`
    - Package built

If so, I could create a follow-up PR to rename those files as well as Makefile and buildkite scripts accordingly.

Reply:
Created PR to rename test scripts: https://github.com/elastic/elastic-package/pull/2072